tasks/insights-client.yml

```diff
@@ -94,6 +94,62 @@
           or "Registered" in __rhc_insights_status.stdout
       changed_when: true
 
+- name: Check display name in insights-client config
+  when:
+    - rhc_insights.display_name is defined
+    - not rhc_insights.display_name is none
+    - rhc_insights.display_name != __rhc_state_absent
+    - rhc_insights.display_name != omit
+  lineinfile:
+    path: "{{ __rhc_insights_conf }}"
+    regexp: "^display_name="
+    state: present
+    line: display_name={{ rhc_insights.display_name }}
+  check_mode: true
+  register: __insights_display_name_exists
+
+- name: Configure display name
+  when:
+    - rhc_insights.display_name is defined
+    - not rhc_insights.display_name is none
+    - rhc_insights.display_name != omit
+    - rhc_insights.display_name != __rhc_state_absent
+    - __insights_display_name_exists.changed
+  block:
+    - name: Update display name in insights-client config
+      lineinfile:
+        path: "{{ __rhc_insights_conf }}"
+        regexp: "^display_name="
+        insertafter: "#display_name="
+        line: display_name={{ rhc_insights.display_name }}
+    - name: Update display namein inventory
```

Comment:
```suggestion
    - name: Update display name in inventory
```

Reply:
Also probably Inventory should be written with a capital I. But keeping that lowercase for consistency with the Ansible Host tasks descriptions.